operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/AbstractEventSourceHolderDependentResource.java

```diff
@@ -75,15 +64,9 @@ private RecentOperationCacheFiller<R> recentOperationCacheFiller() {
     return (RecentOperationCacheFiller<R>) eventSource;
   }
 
-  public void setOnAddFilter(Predicate<R> onAddFilter) {
-    this.onAddFilter = onAddFilter;
-  }
-
-  public void setOnUpdateFilter(BiPredicate<R, R> onUpdateFilter) {
-    this.onUpdateFilter = onUpdateFilter;
-  }
-
-  public void setOnDeleteFilter(BiPredicate<R, Boolean> onDeleteFilter) {
-    this.onDeleteFilter = onDeleteFilter;
+  public void initFilters(Predicate<R> onAddFilter, BiPredicate<R, R> onUpdateFilter,
```

Comment:
When this is intended to be called? Since it should be after`initEventSource`, but that is called after the event source initialized. Can we add an integration test for this?